apps/trading/pages/styles.css

```diff
@@ -12,6 +12,7 @@ body,
 html {
   --focus-border: theme('colors.vega.pink');
   --separator-border: theme('colors.neutral.300');
+  --pennant-color-danger: theme('colors.vega.pink');
```

Comment:
**praise (non-blocking)**:  I didn't realise Pennant supported theming this way, nice work @johnwalley & @MadalinaRaicu

Reply:
Worth pointing out for @johnwalley that it was non obvious that sell stroke color controls both the stroke and the fill. It would probably be clearer to allow the user to define stroke and fill separate for sell candles.